### PR TITLE
Dashboard

### LINKED ISSUE #1: Dashboard
can i make a small dashboard for configuring the bot easier, it would help a lot.

### PR DESCRIPTION
**Pull request by DarkPheonix2122**
its not fully finished, gotta add the plugins and stuff, but its good

### CODE REVIEW

**Comment by DarkPheonix2122** [comment]
Good so far 😁

File: README.md [hunk 1]
```diff
@@ -18,4 +18,4 @@ This TOS is being updated August 15th, 2020 and will require all bots to have a
 
 More info can be found on the [Discord Developers](https://discord.gg/discord-developers) server in [this](https://discord.com/channels/613425648685547541/697489244649816084/728031320625905794) message.
 
-It is your responsibility to make sure your use of this bot is not against the TOS of any services it uses.
\ No newline at end of file
+It is your responsibility to make sure your use of this bot is not against the TOS of any services it uses.
```
Comment (TheShadowGamer) [inline README.md:21]: What is the difference here

File: dashboard/package.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@julianyaman/discord-bot-dashboard",
```
Comment (TheShadowGamer) [inline dashboard/package.json:2]: Wait so you did not make this?

File: dashboard/package.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    "Huskydog9988 (GitHub: Huskydog9988)"
+  ],
+  "homepage": "https://github.com/julianYaman/discord-bot-dashboard#readme",
+  "devDependencies": {
+    "dotenv": "^8.2.0",
+    "nodemon": "^2.0.2",
+    "better-sqlite3": "^7.0.1",
+    "node-os-utils": "^1.2.2"
+  },
+  "dependencies": {
```
Comment (TheShadowGamer) [inline dashboard/package.json:30]: These should be in the main package.json

File: dashboard/test.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
+/*****
```
Comment (TheShadowGamer) [inline dashboard/test.js:1]: This file should not exist. The dashboard part should be included in the main index.js.

**Review by TheShadowGamer — CHANGES_REQUESTED** [review]
I’m just now reviewing this PR.
I have these questions/changes, then I will continue to review the PR.

**Comment by TheShadowGamer** [comment]
Seeing as ive had no response from @DarkPheonix2122, I’m going to close this.

**Pull request closed without merging**